lib/generators/templates/models/version.rb

```diff
@@ -0,0 +1,3 @@
+class Version < ActiveRecord::Base
+  include ActiveVersioning::Model
+end
```

Comment:
wonder if there are any code snippets you could add in here that are commented out for common additional functionality